pkg/clipboard/clipboard.go

```diff
@@ -16,27 +17,58 @@ var (
 	// -ldflags=='-X github.com/gopasspw/gopass/pkg/clipboard.Helpers=termux-api'.
 	Helpers = "xsel or xclip"
 	// ErrNotSupported is returned when the clipboard is not accessible.
-	ErrNotSupported = fmt.Errorf("WARNING: No clipboard available. Install " + Helpers + " or use -f to print to console")
+	ErrNotSupported = fmt.Errorf("WARNING: No clipboard available. Install " + Helpers + ", provide $GOPASS_CLIPBOARD_COPY_CMD and $GOPASS_CLIPBOARD_CLEAR_CMD or use -f to print to console")
 )
 
 // CopyTo copies the given data to the clipboard and enqueues automatic
 // clearing of the clipboard.
 func CopyTo(ctx context.Context, name string, content []byte, timeout int) error {
-	if clipboard.Unsupported {
-		out.Printf(ctx, "%s", ErrNotSupported)
-		_ = notify.Notify(ctx, "gopass - clipboard", fmt.Sprintf("%s", ErrNotSupported))
-		return nil
-	}
+	clipboardCopyCMD := os.Getenv("GOPASS_CLIPBOARD_COPY_CMD")
+	if clipboardCopyCMD != "" {
```

Comment:
This function is getting too big to comprehend quickly.

Please turn the if into a single line statement, e.g. `if cmd := os.Getenv("..."); cmd != "" {`and then call into another function, e.g. `if err := copyToCmd(ctx, ..., cmd);`.

Reply:
Fixed in a0986a93